### DISCUSSION TITLE
Slight delay when checking the checkbox

### DISCUSSION THREAD

**Discussion by DrOreo002** [rappasoft/laravel-livewire-tables#483] [Q&A]
Hey, I recently got the same problem as this guy https://github.com/rappasoft/laravel-livewire-tables/issues/266 and it seems like its livewire model's thing that causes it and then by looking at the docs there's a defer method so I have an idea if maybe there's any way so that I can defer the checkbox value and only send it when the bulk action button is pressed?

**Comment by rappasoft** [comment]
I only get a slight delay on the first one while it renders the bulk row. But everything from there is near-instant for me, this is locally on Laravel Valet so a production environment should be much faster.

https://user-images.githubusercontent.com/6026504/136665057-11d3380a-17c4-43bd-9b13-764370c44752.mov

**Reply by DrOreo002** [reply]
Hmmm I did some debugging and it seems like checking the box calls query method? if that's so then that might what cause mine to lags, since I done a lot of filtering there.
